docs/verification.md

```diff
@@ -0,0 +1,71 @@
+# Verifying Trusted Setup Ceremony
+
+A trusted setup ceremony is a fundamental process in ensuring the integrity and security of certain cryptographic systems. 
+By generating a unique and unpredictable “number” collectively, with contributions from multiple parties, 
+we create a foundation of trust that prevents malicious actors from compromising the system.
+
+Verifying the security of our ceremony is paramount. However, it's important to note that it's a more technical process than contributing. 
+We invite everyone to confirm the security of our ceremony for themselves, and so are publishing this guide to help you do so.
+
+
+## Overview
+There are multiple steps to verifying the security of a trusted setup ceremony. We provide the required list in a way 
+that malicious projects pretend to run rituals while secretly retaining backdoors. 
+To ensure that anyone can verify the integrity of our trusted setup ceremony, we have outlined a robust guide for 
+verifying the results. Here’s an overview of the key steps involved:
+
+- Verifying the circuit we are securing does not include security holes;
+- Verifying the transcript matches the circuit;
+- Verifying contributors' attestations match the transcript;
+
+## Verifying the Circuit
+Confirming the integrity and security of the [EthStorage blob poseidon circuit](https://github.com/ethstorage/zk-decoder/blob/main/circom/circuits/blob_poseidon_2.circom) 
```

Comment:
`https://github.com/ethstorage/zk-decoder` is a private repo, not sure everyone can see the code